entrypoint.sh

```diff
@@ -3,7 +3,13 @@
 cd "$GITHUB_WORKSPACE"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-git ls-files --exclude='Dockerfile*' --ignored | xargs hadolint \
+EXCLUDES=""
+for exclude_path in $INPUT_EXCLUDE; do
+  EXCLUDES="EXCLUDES --exclude='!$exclude_path'"
```

Comment:
```suggestion
  EXCLUDES="$EXCLUDES --exclude='!$exclude_path'"
```
Is the dollar sign optional? Will this work without it?

Reply:
No, that's right, I missed this one.